src/bridge/PublicitySyncer.ts

```diff
@@ -126,25 +122,27 @@ export class PublicitySyncer {
         const mappings = await this.ircBridge.getStore().getMatrixRoomsForChannel(server, channel);
         const roomIds = mappings.map((m) => m.getId());
 
-        this.visibilityMap.mappings = {};
+        this.visibilityMap.mappings = new Map();
 
         // Update rooms to correct visibilities
-        let currentStates: {[roomId: string]: "public"|"private"} = {};
+        let currentStates: Map<string, "public"|"private"> = new Map();
```

Comment:
Surprised we don't have a type for "public"|"private"

Reply:
Added in https://github.com/matrix-org/matrix-appservice-irc/pull/1614/commits/7f58bfc1204703ee1b0d38d8dac215c1539764d5